R/functions_estimation.R

```diff
@@ -307,19 +293,17 @@ estimate.formula <- function(
 
   ### check model and subModel
   checkModelPar(model, subModel,
-    modelList = c("DyNAM", "REM", "DyNAMi", "TriNAM"),
+    modelList = c("DyNAM", "REM", "DyNAMi"),
     subModelList = list(
       DyNAM = c("choice", "rate", "choice_coordination"),
       REM = "choice",
-      DyNAMi = c("choice", "rate"),
-      TriNAM = c("choice", "rate")
+      DyNAMi = c("choice", "rate")
     )
   )
 
   stopifnot(
     inherits(preprocessingOnly, "logical"),
     inherits(verbose, "logical"),
-    inherits(silent, "logical"),
     inherits(debug, "logical"),
```

Comment:
Apparently `debug()` is undesirable...

Reply:
The easiest is to change the name of the argument. We already use `verbose` for printing the messages during estimation.
`debug` prints the coefficients, log-likelihood, and score change after every iteration. Any suggestions for a better name

Reply:
Why not `messages` and `verbose`, or `verbose` and `updates`?

Reply:
`?optimHess` uses `trace`. What about `verbose` and `trace`?

Reply:
trace could work, but could be confusing if we then talk about trace diagnostics? Another option for the former would be `progress`?

Reply:
Done in [efb815f](https://github.com/snlab-ch/goldfish/pull/77/commits/efb815fde17ae6643e98bc282790cef0bd725438)